pyo3-ffi/src/object.rs

```diff
@@ -507,8 +557,13 @@ extern "C" {
 
 #[inline(always)]
 pub unsafe fn Py_INCREF(op: *mut PyObject) {
```

Comment:
Potentially can we leave a TODO or fixme to use inline reference counting on freethreaded builds? (Here and in `Py_DECREF`.)

If it helps us eke out a tiny bit of performance, it might be a nice thing to do eventually.